ts/features/profileSettings/store/reducers/index.ts

```diff
@@ -0,0 +1,43 @@
+import { getType } from "typesafe-actions";
+import { PersistConfig, persistReducer } from "redux-persist";
+import AsyncStorage from "@react-native-async-storage/async-storage";
+import { setShowProfileBanner } from "../actions";
+import { Action } from "../../../../store/actions/types";
+
+export type ProfileSettingsState = {
+  showProfileBanner: boolean;
+};
+
+export const profileSettingsReducerInitialState = {
```

Comment:
Usually i use this naming pattern for global constants, since this constant is used only in this file i preferred to use camelCase. Rather, what do you think about removing export?

Reply:
Okay, right. So I guess we can remove `export` for now. What do you think?